fasta/faBin.go

```diff
@@ -0,0 +1,250 @@
+package fasta
+
+import (
+	"github.com/vertgenlab/gonomics/dna"
+	"log"
+)
+
+//BinGenomeNoBreaks takes in an entire genome which is sorted largest to smallest contig and breaks up the fasta so
+//that smaller contigs get combined into a single fasta, while large contigs become a single fasta on their own. The user
+//must specify the number of bins for the genome to be broken into, the genome must have more contigs than bins in order
+//to combine any contigs and equal number bins to contigs if each contig gets its own record. The bins will all be
+//filled with the first contig encountered when it's empty, and then the smallest of those bins will be filled when the
```

Comment:
Suggestion for language change... 
instead of "smallest bin" calling it "the bin with the smallest contig"
Can talk more at code review.